external_server/external_server.py

```diff
@@ -116,21 +122,22 @@ def _init_seq_connect(self) -> None:
         devices = received_msg.connect.devices
         for device in devices:
             if device.module not in self._modules:
-                self._logger.error(f"Module {device.module} not supported")
+                self._logger.warning(f"Module {device.module} not supported, cannot complete connect sequence with current configuration")
```

Comment:
I don't think that the connection sequence should fail if the module is not supported. I would rather log it, allow connection and discard statuses and respond with an empty command in the connection sequence. We should add more return codes to status response like: module not supported, device not supported etc and in this case respond with the module not supported response. Right now log the warning and continue, we will create task for the additional error codes.